types/ConnectionStatus.d.ts

```diff
@@ -1,4 +1,4 @@
-export declare interface GetConnectionStatusParameters {
+export interface GetConnectionStatusParameters {
```

Comment:
Maybe all `types/*` stuff can be moved to separate `@types/webostv` typings project.
It would be helpful for developers who use webostv.js with typescript